frontend/src/app/app.module.ts

```diff
@@ -17,6 +20,10 @@ import { CampaignsComponent } from './campaigns/campaigns.component';
 import { GuideComponent } from './guide/guide.component';
 import { AnalyticsComponent } from './analytics/analytics.component';
 import { IntroComponent } from './intro/intro.component';
+import {MatTableModule} from '@angular/material/table';
+
+//fixing 404 problem here by using # pr
+import { HashLocationStrategy, LocationStrategy  } from '@angular/common';
```

Comment:
Yeah, I was looking into this as well. Seems like this should fix it. We will also need to import `Location` to [normalize](https://angular.io/api/common/HashLocationStrategy) any paths.

Reply:
Alright, I think we will be going with your fix for this line when merging.